File: 2026/1-semestre/fase-4/atividade/android-estoque/app/src/main/java/com/fiap/vinheriaestoque/data/Produto.kt
package com.fiap.vinheriaestoque.data

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "produtos")
data class Produto(
    @PrimaryKey(autoGenerate = true)
    val id: Int = 0,
    val nome: String,
    val tipo: String,
    val safra: Int,
    val quantidade: Int,
    val preco: Double,
    val origem: String,
    val descricao: String,
)

File: 2026/1-semestre/fase-4/atividade/csharp-estoque/Models/Produto.cs
namespace VinheriaEstoque.Models;

public sealed class Produto
{
    public int Id { get; set; }
    public string Nome { get; set; } = string.Empty;
    public string Tipo { get; set; } = string.Empty;
    public int Safra { get; set; }
    public int Quantidade { get; set; }
    public decimal Preco { get; set; }
    public string Origem { get; set; } = string.Empty;
    public string Descricao { get; set; } = string.Empty;
}
